html/elements/link.json

```
@@ -732,7 +733,8 @@
                   "notes": "In version 79 and later (Blink-based Edge), if both ICO and PNG are available, will use ICO over PNG if ICO has better matching sizes set. (Per caniuse.com.)"
                 },
                 "firefox": {
-                  "version_added": "2"
+                  "version_added": "2",
+                  "notes": "Prior to Firefox 83, the crossorigin attribute was not supported for rel=icon"
```

suggestion: nitpicky style stuff

```suggestion
                  "notes": "Before Firefox 83, the <code>crossorigin</code> attribute is not supported for <code>rel=\"icon\"</code>."
```